app/src/pages/feedbackPage/FeedbackPage.test.tsx

```diff
@@ -1,50 +1,284 @@
+            const mockInputData = {
+                feedbackContent: 'Mock feedback content',
+                respondentName: 'Jane Smith',
+                respondentEmail: 'jane_smith@testing.com',
+            };
+
+            renderComponent();
+
+            act(() => {
+                fillInForm(mockInputData);
+                clickSubmitButton();
+            });
+
+            await waitFor(() => {
+                expect(screen.getByText('Please select an option')).toBeInTheDocument();
+            });
+            expect(mockedAxios).not.toBeCalled();
+            expect(screen.queryByText('Submitting...')).not.toBeInTheDocument();
+        });
+
+        it("on submit, if user filled in an invalid email address, display an error message and don't send email", async () => {
+            const mockInputData = {
+                feedbackContent: 'Mock feedback content',
+                howSatisfied: SATISFACTION_CHOICES.VerySatisfied,
+                respondentName: 'Jane Smith',
+                respondentEmail: 'some_random_string_1234',
+            };
+
+            renderComponent();
+
+            act(() => {
+                fillInForm(mockInputData);
+                clickSubmitButton();
+            });
+
+            await waitFor(() => {
+                expect(screen.getByText('Enter a valid email address')).toBeInTheDocument();
+            });
+            expect(mockedAxios).not.toBeCalled();
+            expect(screen.queryByText('Submitting...')).not.toBeInTheDocument();
+        });
+
+        it('on submit, allows the respondent name and email to be blank', async () => {
+            mockedAxios.post.mockImplementation(() =>
+                Promise.resolve({ status: 200, data: 'Success' }),
+            );
+            const mockInputData = {
+                feedbackContent: 'Mock feedback content',
+                howSatisfied: SATISFACTION_CHOICES.VeryDissatisfied,
+            };
+            const expectedEmailContent = {
+                ...mockInputData,
+                respondentName: '',
+                respondentEmail: '',
+            };
+
+            renderComponent();
+
+            act(() => {
+                fillInForm(mockInputData);
+                clickSubmitButton();
+            });
+
+            await waitFor(() =>
+                expect(mockedAxios.post).toBeCalledWith(
+                    baseURL + '/Feedback',
+                    expectedEmailContent,
+                    {
+                        headers: {},
+                    },
+                ),
+            );
+            expect(screen.getByText('Submitting...')).toBeInTheDocument();
+            expect(screen.getByRole('button')).toHaveAttribute('disabled');
+            expect(mockedUseNavigate).toHaveBeenCalledWith(routes.FEEDBACK_CONFIRMATION);
+        });
+    });
+
+    describe('Navigation', () => {
```

Comment:
In this block, we should add a unit test for navigating to the confirmation page when the feedback endpoint returns 200.